src/transaction/log_page_buffer.c

```diff
@@ -1913,6 +1913,11 @@ logpb_copy_page (THREAD_ENTRY * thread_p, LOG_PAGEID pageid, LOG_CS_ACCESS_MODE
   else
     {
       er_set (ER_FATAL_ERROR_SEVERITY, ARG_FILE_LINE, ER_LOG_PAGE_CORRUPTED, 1, pageid);
+      if (log_csect_entered)
+        {
+          rv = ER_LOG_PAGE_CORRUPTED;
+          goto exit;
+        }
       return ER_LOG_PAGE_CORRUPTED;
```

Comment:
```suggestion
      rv = ER_LOG_PAGE_CORRUPTED;
      goto exit;
```

`log_csect_entered` check and error code return is not required here, because it is done redundantly at `exit` label.

Reply:
@joohok 
Thank you. I got it.
I have changed my commit again.